utils/swift_build_support/swift_build_support/arguments.py

```diff
@@ -200,10 +201,30 @@ def __init__(self,
             nargs="?",
             type=type.bool,
             help=help,
-            const=True)
+            const=const)
 
     def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, values)
 
 
 _register(action, 'optional_bool', _OptionalBoolAction)
+
+
+class _OptionalTrueAction(_OptionalBoolAction):
```

Comment:
Can you add documentation for this and the other option?

Reply:
Sure thing! I don't expect these classes and really this entire module to hang around for very long, I just needed this particular behavior sooner than later.